src/frontend/lib/views/insights/widgets/bottom_time_sheet.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:frontend/models/field_model.dart';
+import 'package:frontend/models/scan_model.dart';
+import 'package:frontend/providers/field_scan_provider.dart';
+import 'package:frontend/stores/scan_store.dart';
+import 'package:intl/intl.dart';
+import 'package:provider/provider.dart';
+
+class BottomTimeSheet extends StatefulWidget {
+  const BottomTimeSheet({
+    super.key,
+    required this.fieldModel,
+  });
+  final FieldModel fieldModel;
+  @override
+  State<BottomTimeSheet> createState() => _BottomTimeSheetState();
+}
+
+class _BottomTimeSheetState extends State<BottomTimeSheet> {
+  @override
+  Widget build(BuildContext context) {
+    return DraggableScrollableSheet(
+      builder: (context, scrollController) {
+        var selectedFieldScan = Provider.of<FieldScanProvider>(context).selectedFieldScan;
+        return selectedFieldScan == null ? const SizedBox.shrink() : Container(
+                padding: const EdgeInsets.only(top: 10, left: 20, right: 20),
+                decoration:  BoxDecoration(
+                  color: Colors.white.withOpacity(0.8),
+                  borderRadius: const BorderRadius.only(
+                    topLeft: Radius.circular(20),
+                    topRight: Radius.circular(20),
+                  ),
+                ),
+                child: ListView(
+                  controller: scrollController,
+                  children: <Widget>[
+                    Center(
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          // Button to change selected field scan
+                          IconButton(
+                            icon: const Icon(Icons.arrow_back_ios),
+                            onPressed: Provider.of<FieldScanProvider>(context, listen: true).isFirstFieldScan()? null : () {
+                              Provider.of<FieldScanProvider>(context, listen: false).selectPreviousFieldScan();
+                            },
+                          ),
+                          Text(
+                            DateFormat('dd-MM-yyyy').format(selectedFieldScan.endDate),
+                            style: const TextStyle(fontSize: 20, fontWeight: FontWeight.bold, fontFamily: 'Roboto'),
```

Comment:
```suggestion
                            style: const TextStyle(fontSize: 20, fontWeight: FontWeight.bold'),
```

A bit nitpicking here perhaps, but keeping the default font makes the view a bit more consistent imo. Wdyt?

![image](https://user-images.githubusercontent.com/74115586/224507945-c3ea77f2-60ef-4d15-a4ba-f7a35faa99b9.png)

Reply:
I personally dont really like the default font we are using

Reply:
Ahahah fair enough, I guess let's change the root font set in `main.dart` then?

Reply:
Yup, will add it here

Reply:
I feel like the current font is quite heavy, which one did you change it to?

Reply:
I was just thinking Roboto or maybe OpenSans.
https://fonts.google.com/specimen/Open+Sans

Reply:
Sounds good to me yeah :)

Reply:
OpenSans seems to make the app look very nice. I think we should go with this one